src/query/settings/src/settings_default.rs

```diff
@@ -531,6 +531,12 @@ impl DefaultSettings {
                     possible_values: None,
                     mode: SettingMode::Both,
                 }),
+                ("query_flight_compression", DefaultSettingValue {
+                    value: UserSettingValue::String(String::from("None")),
```

Comment:
From my test using this image(https://github.com/datafuselabs/databend/pull/13934#issuecomment-1844108465), `query_flight_compression` default value is Lz4 not None, weird.